src/TritonCTS/src/HTreeBuilder.cpp

```diff
@@ -287,47 +285,45 @@ void HTreeBuilder::computeLevelTopology(unsigned level, double width, double hei
         for (unsigned charSegLength = _techChar->getMaxSegmentLength(); charSegLength >= 1; --charSegLength) {
                 unsigned numWires = (segmentLength - currLength) / charSegLength;
                 
-                if (numWires < 1) {
-                        continue;
-                }
-
-                currLength += numWires * charSegLength;
-                for (unsigned wireCount = 0; wireCount < numWires; ++wireCount) {
-                        unsigned outCap = 0, outSlew = 0;
-                        unsigned key = 0;
-                        if (_options->isSimpleSegmentEnabled()){
-                                remainingLength = remainingLength - charSegLength;
-                                
-                                if (segmentLength >= vertexBufferLength && (wireCount + 1 >= numWires) && _options->isVertexBuffersEnabled()){
-                                        remainingLength = 0;
-                                        key = computeMinDelaySegment(charSegLength, inputSlew, inputCap, 
-                                                                SLEW_THRESHOLD, INIT_TOLERANCE, outSlew, outCap, true, remainingLength);
-                                        remainingLength = remainingLength + _options->getBufferDistance() / (_techChar->getLengthUnit());
-                                } else {
-                                        if (remainingLength <= 0){
+                if (numWires >= 1) {
+                        currLength += numWires * charSegLength;
+                        for (unsigned wireCount = 0; wireCount < numWires; ++wireCount) {
+                                unsigned outCap = 0, outSlew = 0;
```

Comment:
always baffled me how a capacitance or slew could be unsigned. what units would that be? I have a note to myself to ask you for a comment at least because it is so far from obvious how either could be an integer

Reply:
There was one discussion on TritonCTS about this same characteristic. The capacitance and slew values are normalized before saving them in the LUT. I plan to change this in the future, but it would be a bigger change.
TritonCTS also reports these normalized values, which doesn't help the user at all...